pkg/controller/directvolumemigration/rsync.go

```diff
@@ -711,6 +711,10 @@ func (t *Task) areRsyncRoutesAdmitted() (bool, []string, error) {
 					message = condition.Message
 					break
 				}
+				if condition.Type == routev1.RouteAdmitted && condition.Status == corev1.ConditionTrue {
+					admitted = true
```

Comment:
If it is admitted do we not have a message on the condition or is the message set somewhere else?

Reply:
no if admitted the Warn condition will be gone. the message is how the check happens a few lines below https://github.com/konveyor/mig-controller/pull/921/files#diff-7299a3c4e600e2733571d1d39c64bbfe6c7e448a6bb15aa472514a173d3095c0R724